desktop/conf/pseudo-distributed.ini.tmpl

```diff
@@ -159,6 +159,7 @@
   # Help improve Hue with anonymous usage analytics.
   # Use Google Analytics to see how many times an application or specific section of an application is used, nothing more.
   ## collect_usage=true
+  ## gtag_id='G-25K7599S1Q'
```

Comment:
Few questions:
1. What is this `gtag_id`? Can this be some sensitive information which we should not expose in our config?
2. We are adding this default `gtag_id` for every Hue instance unless user add customised one?
3. How can the user disable Google Analytics usage collection?

Suggestion: All config changes in `pseudo-distributed.ini.tmpl` (development config template) are added in `hue.ini` (production config) also.

Reply:
1. gtag_id is no secret, it used to be hardcoded, with this change it's configurable.
2. Yes. The default one is ours, the user has the ability to provide their own.
3. by setting collect_usage=false